Gemfile.lock

```diff
@@ -1,7 +1,7 @@
 PATH
   remote: .
   specs:
-    auth-armor (0.1.0)
+    auth-armor (0.1.3)
```

Comment:
Please run bundle install again to remove this gem because it is the gem we're creating here, hence should not be in the gemfile